pyproject.toml

```diff
@@ -82,7 +77,10 @@ doc = [
     "myst_parser",
     "sphinx_design>=0.5.0",
     "readthedocs-sphinx-search",
+    # for unreleased changes
+    "anndata[dev-base,dev-doc]",
```

Comment:
Why `dev-base` as well?

Reply:
good point, I thought we need setuptools-scm, but that’s just for runtime `__version__`